src/main/scala-2.11/markets/orders/orderings/TimeOrdering.scala

```diff
@@ -23,7 +23,7 @@ trait TimeOrdering[T <: Order] extends Ordering[T] {
   def compare(order1: T, order2: T): Int = {
     if (hasTimePriority(order1, order2)) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1508237013/issues/source?bid=2921361&fileBranchId=3005746#l24)